### DISCUSSION TITLE
Adding a rails-like destroy option to the cli?

### DISCUSSION THREAD

**Discussion by lhocke** [blitz-js/blitz#3362] [Ideas] [answered]
I'm definitely liking what I've seen of Blitz so far, but I'd love to see something similar to `rails destroy` and its alias `rails d` added to the cli. Right now removing a model requires editing the schema and running a new migration which is fine but a little clunky and adds extra steps when you might want to remove everything related to it. Having a way built into the cli to remove anything made by `blitz generate` would be really convenient.

**Comment by beerose** [comment, marked as answer]
That's a good idea! I'll convert it to an issue
